pandas/tests/indexes/datetimes/test_tools.py

```diff
@@ -715,6 +716,14 @@ def test_to_datetime_unprocessable_input(self):
         )
         self.assertRaises(TypeError, to_datetime, [1, '1'], errors='raise')
 
+    @pytest.mark.parametrize('arg', [int(1), float(1), range(5),
```

Comment:
move this to a separate class that inherits from ``object`` (to avoid using ``staticmethod``).